cli/operator/node.go

```diff
@@ -784,3 +788,30 @@ func startMetricsHandler(logger *zap.Logger, db basedb.Database, port int, enabl
 		logger.Panic("failed to serve metrics", zap.Error(err))
 	}
 }
+
+func initSlotCleanup(ctx context.Context, logger *zap.Logger, stores *ibftstorage.ParticipantStores, slotTickerProvider slotticker.Provider, retain int) {
+	var wg sync.WaitGroup
+
+	ticker := slotTickerProvider()
+	<-ticker.Next()
```

Comment:
this should be easier to get using `networkConfig.Beacon.EstimatedCurrentSlot` also non blocking. I don't see a point waiting a slot to calculate the threshold.